bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/internal/i18n/I18nConfigOptionsProvider.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.config.core.internal.i18n;
+
+import java.net.URI;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.Locale;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import org.eclipse.smarthome.config.core.ConfigOptionProvider;
+import org.eclipse.smarthome.config.core.ParameterOption;
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * {@link ConfigOptionProvider} that provides a list of
+ *
+ * @author Simon Kaufmann - initial contribution and API
+ *
+ */
+@Component(immediate = true)
+public class I18nConfigOptionsProvider implements ConfigOptionProvider {
+
+    @Override
+    public Collection<ParameterOption> getParameterOptions(URI uri, String param, Locale locale) {
+        if (uri.toString().equals("system:i18n")) {
+            Locale translation = locale != null ? locale : Locale.getDefault();
+            if (param.equals("language")) {
```

Comment:
Oh, the contract in fact is pretty clear:
[ConfigDescriptionParameter.java#L204](https://github.com/eclipse/smarthome/blob/7b9d32430384f1f0bb06381014394b090d8681cb/bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/ConfigDescriptionParameter.java#L204), [config-description-1.0.0.xsd#L59](https://github.com/eclipse/smarthome/blob/78b6ef19d9fc8da7969f41228d3f3b56f21b8a7f/bundles/config/org.eclipse.smarthome.config.xml/config-description-1.0.0.xsd#L59)
It's just at the wrong place/not replicated everywhere (yet), depending on your perspective.